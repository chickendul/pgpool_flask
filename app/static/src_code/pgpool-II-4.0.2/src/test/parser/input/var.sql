version https://git-lfs.github.com/spec/v1
oid sha256:a0112645a3dfbf8a9ed301c39fdd8700b96547e1e391d685f4a21213016bb371
size 413
