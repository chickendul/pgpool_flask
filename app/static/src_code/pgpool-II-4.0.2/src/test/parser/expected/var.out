version https://git-lfs.github.com/spec/v1
oid sha256:3de7ca8c9e6573c0da0647dfd1f380b70fd6a3de6d35fe184e79606471a486a4
size 409
